.github/workflows/no-text.yml

```
@@ -0,0 +1,32 @@
+name: no-text
+on:
+  - push
+  - pull_request
```

Copied from the `haskell-ci` setup:

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```